wagtailio/project_styleguide/templates/patterns/components/streamfields/code_block/code_block.html

```diff
@@ -1,6 +1,6 @@
 {% load wagtailcore_tags %}
 <div class="grid">
-    <div class="grid-align">
```

Comment:
Renamed because this is the only place `.grid-align` was used, so `.code-block` makes more sense